pkg/volume/plugins.go

```diff
@@ -1028,6 +1041,22 @@ func (pm *VolumePluginMgr) FindNodeExpandablePluginByName(name string) (NodeExpa
 	return nil, nil
 }
 
+// GetCSIKubeletPluginHandler fetches the CSI plugin and retruns its plugin handler
+func (pm *VolumePluginMgr) GetCSIKubeletPluginHandler() (pluginwatcher.PluginHandler, error) {
+	volumePlugin, err := pm.FindPluginByName(csiid.CSIPluginName)
+	if err != nil {
+		return nil, err
+	}
+
+	kubeletWatchablePlugin, ok := volumePlugin.(KubeletWatchableVolumePlugin)
+	if !ok {
+		// TODO (hoegaarden) Can this even happen?
```

Comment:
Yes, this can happen in a case where code changes and no longer satisfies the interface.

Reply:
I think, that cannot really happen, because we have the compile time assertion here: https://github.com/kubernetes/kubernetes/pull/74966/files/7b30c10ec5ba1578c5306df33722c9fb188d7f31#diff-f52821ec6236e86b02ca334533395c91R1015

But someone could remove that somewhen, so we def. need to check the type here.

In any case, I will remove the comment.